bin/studio.py

```diff
@@ -0,0 +1,156 @@
+from ensurepip import version
+import argparse
+import csv
+import sys
+import json
+
+from setuptools import Command
+
+# This script will implement cli for Studio public API.
+# Originally wanted to mimick aws cli with
+# "studio media show" like syntax, but aws cli heavily
+# customizes argparse, see
+# https://github.com/aws/aws-cli/blob/45b0063b2d0b245b17a57fd9eebd9fcc87c4426a/awscli/argparser.py
+
+from utils.utils import PublicAPIClient, get_commandline_arguments
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
I would even drop '--subdomain' selector from here and just have this argument as a positional one.